torch/utils/hipify/hipify_python.py

```diff
@@ -65,7 +64,7 @@ def __str__(self):
            'preprocess_file_and_save_result', 'compute_stats', 'add_dim3', 'processKernelLaunches', 'find_closure_group',
            'find_bracket_group', 'find_parentheses_group', 'replace_math_functions', 'hip_header_magic', 'replace_extern_shared',
```

Comment:
`replace_extern_shared` definition should be marked deprecated and the following lines should be deleted: https://github.com/pytorch/pytorch/blob/765a9fd59afa97ece0b9a1546a2da5410eb87fa1/torch/utils/hipify/hipify_python.py#L907